java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/ActivityResponse.java

```diff
@@ -8,4 +8,7 @@
 @NoArgsConstructor
 public class ActivityResponse extends FCResponse {
   Double emails;
+  Double online;
+  Double social;
+  Double employment;
```

Comment:
This all matches bouncer class ActivityResponse, except bouncer has added a new field "message."